transcom/mymove
# B 18085 identify create values 7 12

## B-18085

## Summary

MilMove needs to provide the PPM Closeout Counselor with a complete view/listing of PPM Costing Calculations upon PPM Closeout.

This PR handles values 7-12:
1. Gross Incentive
2. Government Constructive Cost(GCC)
3. Advance Operating Allowance(AOA)
4. Remaining Incentive
5. Linehaul/short haul price

## Verification Steps for Reviewers

These are to be checked by a reviewer.


### How to test

1. Use any PPM move that has a sit or use testharness "MoveWithPPMShipmentReadyForFinalCloseoutWitSIT"*
2. Log in as a TOO and find that move
3. Review and accept all docs
4. Log in as user for that move
5. Upload documents
6. Set the PPM move status in the DB to "NEEDS_PAYMENT_APPROVAL"
7. In postman go to the Get ppmCloseout endpoint and use that move's PPM ID as the "ppmShipmentID"
8. Verify that fields 7-12 are populated

***TestHarnessData:**
 locator = "66K9XY" ppmshipmentId ="1c39a0f5-26eb-4d24-998a-d109de0c980e"

## Screenshots
<img width="971" alt="Screenshot 2024-01-29 at 7 53 02 AM" src="https://github.com/transcom/mymove/assets/146969726/097e0e58-4c46-41d2-b43e-1f7caaafed2d">
